app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -489,18 +513,17 @@ class BrowserTabFragment :
     private fun showHome() {
         errorSnackbar.dismiss()
         newTabLayout.show()
+        browserLayout.gone()
```

Comment:
It's related.
- `newTabLayout` and `browserLayout` are siblings on the same view (and they can't be visible at the same time). 
- Both contain different views inside of them. We were showing/hiding child views inside `showHome`/`showBrowser` instead of just showing/hiding their parent views.
- As an example, for the new tab, we were only showing/hiding the background logo, and not handling correctly the visibility of other views (e.g: widget cta, favorites, etc.) -> Found a related bug to that.
- Just hiding/showing parents makes easier the handling of those 2 possible states.